core/src/main/java/nl/nn/adapterframework/util/Dir2Xml.java

```diff
@@ -63,13 +62,24 @@ public String getDirList(boolean includeDirectories, int maxItems) {
 				dirXml.addSubElement(getFileAsXmlBuilder(parent,".."));
 			}
 		}
+		
+		int numberOfDirectories = 0;
 		for (int i = 0; i < count; i++) {
 			File file = files[i];
 			if (file.isDirectory() && !includeDirectories) {
+				numberOfDirectories++;
 				continue;
 			}
 			dirXml.addSubElement(getFileAsXmlBuilder(file,file.getName()));
 		}
+		
+		// TODO: implement includeDirectories on WildCardFilter
```

Comment:
I remember discussing this particular problem and like the solution, you could however document it a little better next time :).